### ISSUE TITLE
Get error message from AGOL

### ISSUE THREAD

**Issue by jacobdadams** [agrc/palletjack#77]
If we get an odd error 500, create something that grabs the token and job id and returns the error automatically rather than requiring the end user to do it manually.

**Comment by steveoh** [comment]
Will you elaborate on this a bit @stdavis or @jacobdadams since I'm unfamiliar with the issue? What error messages are we having trouble accessing? What actions are the job ids linked to? What is the manual process for getting the error message?

**Comment by jacobdadams** [comment]
Yeah, it's pretty well spelled out near the bottom of the [readme](https://github.com/agrc/palletjack#troubleshooting-weird-append-errors)

**Comment by stdavis** [comment]
I would be great if this was generic enough to be useable in other projects such as forklift.

**Comment by jacobdadams** [comment]
Agreed. Spitballing here, I wonder if writing it as a decorator makes sense. I'll look into that when I've got some prof dev time.

Also, again spitballing, I wonder if we could modify forklift to use the classes from palletjack for any AGOL work? Bring all our "foo to AGOL" workflows under one codebase?

**Comment by jacobdadams** [comment]
Thought: get token programmatically from arcgis.auth module

**Comment by crosner13** [comment]
Hey @jacobdadams, I was the one who was talking about grabbing a token from the GIS object today!
```
from arcgis.gis import GIS  
gis = GIS("https://www.arcgis.com", "your_username", "********")
token = gis._con.token
print(token)
```
This is how I will sometimes access the token via the GIS object. Alternatively, I'll access it with `requests` and a call to the REST API.

**Comment by jacobdadams** [comment]
Awesome, thanks Cole!
